docs/eloquent-models/relationships.txt

```diff
@@ -0,0 +1,24 @@
+.. _laravel-eloquent-model-relationships:
+
+============================
+Eloquent Model Relationships
+============================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, odm, code example
+
+Relationships
+-------------
+
+This section describes each of the Laravel Eloquent and MongoDB-specific 
+relationships that you can use and provides examples for hasOne, hasMany, 
+belongsTo, belongsToMany, embedsOne, and embedsMany.
+
+Each example will show, when applicable,  how to define the relationships, 
+access the related models, save/create, destroy/delete, and associate/disassociate.
```

Comment:
Do you want to create a distinct page for hybride relations (mongodb <-> sql)?

Reply:
Do you have any info that could help guide me on hybrid relations? I think it could be good to add as a section on this page, but if it increases the scope, it could be ticketed separately.

Reply:
There is tests that can help: https://github.com/mongodb/laravel-mongodb/blob/4.2/tests/HybridRelationsTest.php

And some hybrid models: https://github.com/mongodb/laravel-mongodb/blob/4.2/tests/Models/SqlUser.php